istio/istio.go

```diff
@@ -383,7 +384,34 @@ func (iClient *IstioClient) executeBookInfoInstall(ctx context.Context, arReq *m
 	return nil
 }
 
-// ApplyRule is a method invoked to apply a particular operation on the mesh in a namespace
+func (iClient *IstioClient)executeHipsterShopInstall(ctx context.Context, arReq *meshes.ApplyRuleRequest) error {
+
+	resp, err := http.Get("https://raw.githubusercontent.com/GoogleCloudPlatform/microservices-demo/master/release/istio-manifests.yaml")
+	if err != nil {
+		panic(err)
+	}
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+
+	if err := ioutil.WriteFile("merged.yaml", body, 0644); err != nil {
+		panic(err)
+	}
+	
+		resp, e := http.Get("https://raw.githubusercontent.com/GoogleCloudPlatform/microservices-demo/master/release/kubernetes-manifests.yaml")
+	if e != nil {
+		panic(e)
+	}
+	defer resp.Body.Close()
+	b, err := ioutil.ReadAll(resp.Body)
+	
+	d, err := ioutil.ReadFile("merged.yaml")
+	if err := ioutil.WriteFile("merged.yaml", append(d, b...), 0644); err != nil {
```

Comment:
Here, to the above variable `yamlFileContents` (which you will be creating based on my previous comment) we can add a "\n---\n" (a new line, '---' and a new line again to simulate a yaml separation) and then append the contents of the second file.
Once we have the complete content we will have to do the execution with something like this:
```
if err := iClient.applyConfigChange(ctx, yamlFileContents, arReq.Namespace, arReq.DeleteOp, false); err != nil {
    return err
}
```

Reply:
One other note, whenever you want to handle error, please log the error using `logrus.Error` or `logrus.Errorf` and also return the error.